await-outside-async:15:10:not_async:'await' should be used within an async function
await-outside-async:26:8:func2.inner_func:'await' should be used within an async function
